nbs/docs/Prediction_intervals_in_forecasting_models.ipynb

```diff
@@ -0,0 +1,3012 @@
+{
```

Comment:
<p>I believe this df isn't used anymore</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/mlforecast/pull/196/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='76'/>